index.html

```diff
@@ -86,14 +86,40 @@
 
 
     <!-- About -->
-    <section id="about" class="container">
+    <section id="about" class="container" style="margin-top: 15vh; color: white;">
```

Comment:
Still try to keep the style in the stylesheet, also you can just use Bootstrap classes.